info/portfolio

```diff
@@ -0,0 +1,70 @@
+<html>
+<body>
+	<head>
+		<title>Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+
+	<nav>
+		<a href="about.html">About Me</a> |
+		<a href="portfolio">Portfolio</a>
+	</nav>
+	<h2>Experience</h2>
+					
+	<p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+			
+	<h2>Relevant Experience</h2>
+			
+	<h3>Web Developer, Relaxr <br>2015 - present</h3>
+	<ul>
+		<li>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.   </li>
+	</ul>
+
+	<br>
+
+	<img src="../site/images/relaxr_deliverable.png" alt="Relaxr Deliverable">
+	
+	<br></br>
+	
+	<h3>Web Developer, Startup Matchmaker <br>2015 - present</h3>
+	
+	<ul>
+		<li>Used a design team's wireframes to develop this company's responsive homepage.</li>
+	</ul>
+	
+	<br>
+	
+	<img src="../site/images/startup_matchmaker_deliverable.png" alt="Startup Matchmaker Deliverable">
+	
+	<br></br>
+
+	<h3>Web Developer, CitiPix <br>2015 - present</h3>
+	
+	<ul>
+		<li>Built web app prototype that allows users to store and quickly retrieve photos using keywords.</li>
+	</ul>
+	
+	<br>
+	
+	<img src="../site/images/citipix_deliverable.png" alt="CitiPix Deliverable">
+
+	<br></br>
+	
+	<h2>Education</h2>
+	
+	<ul>
+		<li>General Assembly - FEWD 2015</li>
+		<li>College of Soft Knocks - B.S. 2009</li>
+		<li>School of Hard Knocks - 2005 (Honors)</li>
+	</ul>
+
+	<a href="http://facebook.com/TaylorHiden">
+		<img src="https://www.facebookbrand.com/img/fb-art.jpg" width="30" alt="Facebook">
+	</a>
+	<a href="http://twitter.com/HIDEN_seek53">
```

Comment:
For some reason, this image isn't loading. Looks like your code is right though. Might be a browser rendering problem, but this shows why it's important to include the alt attribute, which you do successfully.